Bixal/move.mil
# Fix pulse marker on locator map + update build script

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- fix pulse marker on locator map.
- Move DB and http client to services to make PPM testable.
- Fix Google APIs call on behat testing.
- Improve front-end sass building.
- Initial work for updating the login banner.
- Update composer installer signature.

## Testing

To verify the changes proposed in this pull request…

1. Pull branch and import changes.
1. Navigate to PPM tool and Locator Maps.
1. Verify everything works as expected.
